sfcasts/ep1/computed.md

```diff
@@ -1,228 +1,192 @@
-Down here, update the class name to `.component`.
+All right! While we're here, I also want to clean things up. Let's refactor our
+`width` away from `style` and do this properly in a class. But before we do that,
+there's one thing I wanted to do earlier that I forgot about. Notice that down
+here in our styles, we have a single class called `sidebar`... and we're actually
+applying that to the outside element of our template. This is great! But sometimes,
+as a convention, when you have a class that is applied to the root element of a
+component, you'll use the generic name `.component`
 
+Down in `style`, update the class name to match.
 
+This won't make any difference, it's just a nice standard when you need a class
+on your outer element.
 
-This won't make any difference. But it makes things so that if we need styles,
-we can always add a class on the root element and have the rest of the styles
-be inside this root `.component` class. You can use it if you want it or not,
-it is not a big deal.
- 
-Now to remove this `width` style here in our root element, let's go back down to our
-style block and inside `.component` let's add a new class called `&.collapsed` and
-set this to `width: 70px`.
+To remove this `width` style in the root element, let's go back down to our
+style block and, inside `.component` add a new class called `&.collapsed` with
+`width: 70px`.
 
 If you're not familiar with this syntax, it means that if this element has both the
-`component` *and* the `collapsed` class, then it will have the `70px` width applied 
-to it. Now the tricky part here is that we need to conditionally add this `collapsed`
-class to our root element, but there's not really a good way to do that.
+`component` *and* the `collapsed` class, then it will get this width.
+
+Back in the template, delete the `style` attribute.
 
-I'll delete my style attribute.
+Now the tricky part here is that we need to conditionally add the `collapsed`
+class to our root element. But there's not really a good way to do that.
 
 ## Conditional :class
 
-So how can we change this so that we can *conditionally* add a class to this array?
 Luckily `Vue` is here again to rescue us! As it turns out, `:class` has another
-superhero syntax that allows us to do just that!
+superhero syntax.
 
-It looks like this: Instead of an array we're going to pass an object where every
-key in the object will be the class name that you want to add and the value will be
-any expression that resolves to `true` or `false`. This will dictate if
-you want that class to be present in real time! Since we always want these three
-classes, we'll say `$style.component: true`, `'p-3': true` and `'mb-5': true`
+It looks like this: instead of an array, pass an object where every
+key in the object will be the class name that you want to add, and the value will be
+true or false for whether or not you want this class. Since we always want these three
```

Comment:
This is inaccurate: The value can be *anything* (like in my other comment). May be say `and the value is anything that boils down to "true" or "false"` or something similar.

Reply:
I know I'm speaking imprecisely here. My hope is that people will know that true/false can be replaced by anything that resolves down to true/false. I'm using true/false as a short way to say "an expression that evaluates to a boolean". I'm partially "getting away with this" because we show using a variable later.